### DISCUSSION TITLE
Desktop space switching on Macos Sequoia when using corfu and lsp hover

### DISCUSSION THREAD

**Discussion by ScriptMang** [d12frosted/homebrew-emacs-plus#731] [General]
I'm getting an  issue where the current desktop space I have emacs in, keeps switching back to the previous one. In particular after every code completion from corfu when emacs is fullscreen. Or when lsp mode shows function definitions on hover. Currently I'm using emacs-plus@29(29.4).  Found a fix where if I remove the ability to set the app to launch on specific desktop, it doesn't happen.  Is this an emacs issue or emacs plus one?

**Comment by d12frosted** [comment]
🤔  You can easiy test it by downloading [Emacs for macOS](https://emacsformacos.com) and testing a bare bones GNU Emacs. Just make sure to download the same version as Emacs+ you are using.

P.S. The conversation is closed, does it mean you figured it out or was it by accident?

**Reply by ScriptMang** [reply]
Yes, and no. What I provided was a work around until I figured out whether it was emacs plus or bare emacs. All I did was change emacs to be able to launch in all desktop spaces. However, I still want to be able to launch emacs in a specific virtual desktop space regardless.

**Reply by ScriptMang** [reply]
I just downloaded GNU Emacs 29.4-1 and it has the same error.

**Reply by ScriptMang** [reply]
I think this bug report I found on the emacs mailing list is related.
ref: https://debbugs.gnu.org/cgi/bugreport.cgi?bug=71744
